<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>811</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="bgwidget" native="true">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>831</width>
     <height>541</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#bgwidget{
background-color: qlineargradient(spread:pad, x1:0.091, y1:0.101636, x2:0.991379, y2:0.977, stop:0 rgba(209, 107, 165, 255), stop:1 rgba(255, 255, 255, 255));}</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>0</y>
      <width>361</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 36pt &quot;Ubuntu&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Fill in your profile</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>70</y>
      <width>621</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 17pt &quot;Ubuntu&quot;;color:rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>Now that you've created an account, please fill in your profile.</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="usernamefeld">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>130</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0,0,0,0);
font: 75 16pt &quot;Ubuntu&quot;;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="firstnamefeld">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>210</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0,0,0,0);
font: 75 16pt &quot;Ubuntu&quot;;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lastnamefeld">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>300</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0,0,0,0);
font: 75 16pt &quot;Ubuntu&quot;;</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEdit">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>120</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 italic 12pt &quot;Ubuntu&quot;;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>110</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Username</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>190</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>First name</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>280</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Last name</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>130</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Birthday</string>
    </property>
   </widget>
   <widget class="QComboBox" name="gender">
    <property name="geometry">
     <rect>
      <x>625</x>
      <y>230</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Male</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Famele</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>230</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Gender</string>
    </property>
   </widget>
   <widget class="QPushButton" name="continue_2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>390</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:20px;
background-color: qlineargradient(spread:pad, x1:0.573, y1:0.289773, x2:1, y2:0, stop:0 rgba(102, 255, 246, 255), stop:0.98 rgba(0, 0, 0, 255), stop:1 rgba(235, 148, 61, 255));
font: 75 16pt &quot;Ubuntu&quot;;
color:rgb(97, 53, 131)</string>
    </property>
    <property name="text">
     <string>Continue</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
